crates/runtime/src/instance.rs

```diff
@@ -462,25 +457,88 @@ impl Instance {
         Layout::from_size_align(size, align).unwrap()
     }
 
+    /// Construct a new VMCallerCheckedAnyfunc for the given function
+    /// (imported or defined in this module) and store into the given
+    /// location. Used during lazy initialization.
+    ///
+    /// Note that our current lazy-init scheme actually calls this every
+    /// time the anyfunc pointer is fetched; this turns out to be better
+    /// than tracking state related to whether it's been initialized
+    /// before, because resetting that state on (re)instantiation is
+    /// very expensive if there are many anyfuncs.
+    #[cold]
```

Comment:
This is probably less cold now to the point that now it's better to remove this.

Reply:
Done!